neo/cm/CollisionModel_load.cpp

```diff
@@ -3653,7 +3653,7 @@ cm_model_t* idCollisionModelManagerLocal::LoadBinaryModelFromFile( idFile* file,
 	assert( model->polygonRefBlocks == NULL || ( model->polygonRefBlocks->next == NULL && model->polygonRefBlocks->nextRef == NULL ) );
 
 	// RB: FIXME
-#if !defined(__x86_64__) && !defined(_WIN64)
+#if !defined(__x86_64__) && !defined(_WIN64) && !defined(__PPC64__)
```

Comment:
if this is a general 64bit problem there are probably better ways to address this?
like
```if( sizeof(void*) == 4 )
{
    assert(...);
    assert(...);
}
```
could even use `if constexpr (sizeof(void*) == 4)` if you don't mind requiring C++17

Reply:
I am not entirely sure, so far I could re-produce that with PPC64BE and PPC64LE. 

Q: Why do we do `!defined(_WIN64)` here? Is it only applicable to non-Windows?

Reply:
only applicable to non-64bit-windows, I guess (maybe MSVC doesn't define `__x86_64__`? no idea)

Reply:
It's true __x86_64__ isn't defined with the VS compiler. I also haven't looked into this for like 8 years. I only recall that those assertions are triggered in 64 bit builds and I'm not sure if it matters without diving deeply in the collision code again.